.nvmrc

```
@@ -0,0 +1 @@
+v20.18.1
```

```suggestion
20

```

lets change this back to a greedy version range (ie. `20.x`) but since nvm doesn't support `x` we can just set this to `"20"` & it should do the same thing.